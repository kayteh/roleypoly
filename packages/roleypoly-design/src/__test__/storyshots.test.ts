import * as path from 'path'
import initStoryshots from '@storybook/addon-storyshots'
// import Adapter from 'enzyme-adapter-react-16'

// enzyme.configure({ adapter: new Adapter() })

initStoryshots({
  configPath: path.resolve(__dirname, '../../.storybook')
})