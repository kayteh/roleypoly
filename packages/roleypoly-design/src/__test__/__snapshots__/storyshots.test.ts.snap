// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button Default 1`] = `
<button
  className="sc-bdVaJa hPnBtV"
  disabled={false}
  onClick={[Function]}
>
  Example
</button>
`;

exports[`Storyshots Button Disabled 1`] = `
<button
  className="sc-bdVaJa hPnBtV"
  disabled={true}
  onClick={[Function]}
>
  Example
</button>
`;

exports[`Storyshots Button Loading 1`] = `
<button
  className="sc-bdVaJa hPnBtV"
  disabled={false}
  onClick={[Function]}
>
  Example
</button>
`;

exports[`Storyshots Button Primary 1`] = `
<button
  className="sc-bdVaJa hPnBtV"
  disabled={false}
  onClick={[Function]}
>
  Example
</button>
`;

exports[`Storyshots Button Secondary 1`] = `
<button
  className="sc-bdVaJa hPnBtV"
  disabled={false}
  onClick={[Function]}
>
  Example
</button>
`;
